### ISSUE TITLE
Subscriber Black- and Whitelisting ?

### ISSUE THREAD

**Issue by hetzge** [crossbario/autobahn-java#444]
I am looking for the Whitelist / Blacklist feature when publishing to a topic (https://crossbar.io/docs/Subscriber-Black-and-Whitelisting/).
Is it correct that this feature is not available in the java client ? Is it planned to implement it ?

**Comment by oberstet** [comment]
Yes, this is currently missing. Would be easy though;) As far as as I see, adding the feature is a couple of lines in two places:

* https://github.com/crossbario/autobahn-java/blob/master/autobahn/src/main/java/io/crossbar/autobahn/wamp/types/PublishOptions.java
* https://github.com/crossbario/autobahn-java/blob/master/autobahn/src/main/java/io/crossbar/autobahn/wamp/Session.java#L885

**Comment by oberstet** [comment]
* exclude_authid
* exclude_authrole
* eligible_authid
* eligible_atuhrole
